tests/update/prepare_system_for_update_tests.pm

```diff
@@ -24,6 +24,8 @@ sub run() {
     assert_screen('xterm-started');
     assert_script_sudo "chown $testapi::username /dev/$testapi::serialdev";
     assert_script_sudo "echo \"download.use_deltarpm = false\" >> /etc/zypp/zypp.conf";
+    # Refresh repodata - we use zypper, as pkcon might not be there. But they share metadata anyway
+    assert_script_sudo "zypper refresh";
```

Comment:
If packagekit is not available then graphical updates have no sense, right? Then we can filter there minimal tests in main.pm (guiupdates_is_applicable)